tests/features/tmgmt/tmgmt_poetry.feature

```diff
@@ -374,3 +374,32 @@ Feature: TMGMT Poetry features
     And the translation request has organisationAuteur "& DG/directorate/unit from which the document comes"
     And the translation request has serviceDemandeur "& DG/directorate/unit of the person submitting the request"
     And the translation request has remarque "Further remarks & comments"
+
+  Scenario: Adding new languages to the ongoing translation request
+    Given I am logged in as a user with the 'administrator' role
```

Comment:
Editor role don't have rights to add and publish translatable content by default.